common/app/model/Formats.scala

```diff
@@ -146,8 +147,31 @@ object PressedContentFormat {
     }
   }
 
-  implicit val sectionIdFormat = Json.format[SectionId]
-  implicit val pillarFormat = Pillar.format
+  implicit val designTypeFormat: Format[DesignType] = new Format[DesignType] {
+    override def reads(json: JsValue): JsResult[DesignType] = json match {
+      case JsString("Article") => JsSuccess(com.gu.contentapi.client.utils.Article)
+      case JsString("Immersive") => JsSuccess(com.gu.contentapi.client.utils.Immersive)
+      case JsString("Media") => JsSuccess(com.gu.contentapi.client.utils.Media)
+      case JsString("Review") => JsSuccess(com.gu.contentapi.client.utils.Review)
+      case JsString("Analysis") => JsSuccess(com.gu.contentapi.client.utils.Analysis)
+      case JsString("Comment") => JsSuccess(com.gu.contentapi.client.utils.Comment)
+      case JsString("Feature") => JsSuccess(com.gu.contentapi.client.utils.Feature)
+      case JsString("Live") => JsSuccess(com.gu.contentapi.client.utils.Live)
+      case _ => JsError(s"Unknown design type: '$json'")
+    }
+    override def writes(dt: DesignType): JsValue = dt match {
+      case com.gu.contentapi.client.utils.Article => JsString("Article")
```

Comment:
You could import `com.gu.contentapi.client.utils._` or `com.gu.contentapi.client.utils` and prefix utils.*

Could also use .toString when reading and writing case objects if you want to avoid introducing strings constants for each

Reply:
I was thinking about using `.toString` but I prefer to use a `match` because in case a new `DesignType` is added to the CAPI client, the compiler would warn there is a missing case. That should draw the attention to whoever is updating CAPI lib to modify this `Format` (especially the `Reads` actually)